source/guides/clear/mixer.rst

```diff
@@ -258,24 +288,22 @@ set to get a smaller kernel image, which will also be faster to load.
 
       $EDITOR ./local-bundles/editors
 
-#. List the bundles in the mix again to confirm removal.
+#. List the bundles in the mix again to confirm removal of :command:`joe`.
 
    .. code-block:: bash
 
       mixer bundle list  --tree
 
-
 #. Build bundles:
 
    .. code-block:: bash
 
       mixer build bundles
 
-   Look in ~/mixer/update/image/<mix version>/full for the full chroot after the
-   :command:`build` command completes.
+#. Build update content.
 
-#. Build update content. Browse to your \http://localhost site and you'll see
-   the web page is now up, but with no update content. Build the update content:
+#. Browse to your \http://localhost site. The web page appears, yet it has
```

Comment:
@mvincerx  - Lines 305-306 should be under the "Build update content." section.  It should not be a separate step.  It should be like this:

#. Build update content.

First, browse to your \http://localhost site. The web page appears, yet it has no update content.  Build the update content:

.. code-block:: bash

....

After that is completed, our \http://localhost site and now you can see the update
   content for mix version 10.

Reply:
Condensing this section. Comment repeats "Build the update content" twice.